$OpenBSD: patch-SevenZipArchive_m,v 1.1.1.1 2011/07/19 13:58:02 sebastia Exp $

Fix the offset of the parsed output from the external binary.

--- SevenZipArchive.m.orig	Wed Apr 27 18:52:27 2011
+++ SevenZipArchive.m	Wed Apr 27 18:53:02 2011
@@ -85,7 +85,7 @@
     NSArray *lines = [string componentsSeparatedByString:@"\n"];
     
     // take out first 8 lines (header) and last 2 lines (footer)
-	lines = [lines subarrayWithRange:NSMakeRange(8, [lines count] - 8)];
+	lines = [lines subarrayWithRange:NSMakeRange(17, [lines count] - 17)];
 	lines = [lines subarrayWithRange:NSMakeRange(0, [lines count] - 3)];
     
     return [self listSevenZipContents:lines];
